HISTORY.md

```diff
@@ -1,5 +1,11 @@
 # Release History - `open-autonomy`
 
+## 0.1.1 (2022-06-22)
+
+Autonomy:
+- Patches `push-all`, `publish` and `fetch` to support service packages
+- Fixes click context issue on CLI tool
+
```

Comment:
Do we want to add release section for packages, since there are going to be a lot of changes I'm afraid this file will be really large in no time.

Reply:
Yes we should start listing these changes. We will be splitting the repo relatively soon so size should not be a concern.